crates/ra_ide_api/src/inlay_hints.rs

```diff
@@ -481,7 +488,7 @@ fn main() {
 }"#,
         );
 
-        assert_debug_snapshot!(analysis.inlay_hints(file_id).unwrap(), @r###"
+        assert_debug_snapshot!(analysis.inlay_hints(file_id, None).unwrap(), @r###"
         [
             InlayHint {
                 range: [311; 315),
```

Comment:
Oh, one more thing! Let's actually add a test that truncation works? Just a simple smoke one!

Otherwise, LGTM!

bors d+